UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -72,16 +85,29 @@ public void AfterTest(ITest test)
 
                 _denyExecutionSynchronizationContext?.ThrowIfSwitchOccurred();
             }
+            catch (Exception ex) when (_threadException != null)
+            {
+                // ignore the follow-up exception
+                LoggingService.Log($"{nameof(AfterTest)} ignoring", ex);
+            }
```

Comment:
💡 It's fine to keep track of both `ex` and `_threadException` when both occur. However, the logging is unnecessary complexity. The exception handler can instead look like this:

```csharp
catch (Exception ex) when (_threadException != null)
{
  // Track both exceptions
  _threadException = new AggregateException(_threadException, ex);
}
```

Reply:
Yes, `AggregateException` is a very good hint.